pkg/provisioner/controller.go

```diff
@@ -351,6 +360,8 @@ func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBuc
 		if err != nil {
 			return fmt.Errorf("error updating OBC %q's status to: %v", v1alpha1.ObjectBucketClaimStatusPhaseBound, err)
 		}
+		obc = updatedOBC // don't let obc become nil but allow it to be updated on success
```

Comment:
this assignment has already been done above and seems unnecessary here.

Reply:
This is for posterity I guess. It makes it possible for someone to add more code after this that uses the `obc`, and there is less risk that they forget to make sure the `obc` is the latest.